### PR TITLE
DEVSU-2244 CD8+ T Cell Percentile Visibility Toggle

### PR DESCRIPTION
**Pull request by bnguyen-bcgsc**
- DEVSU-2244
- Add percentileHidden to ImmuneType
- Add visibility toggle for CD8+ percentile in tumour summary edit
- Enforce visibility rule for CD8+ percentile on genomic and rapid summaries
- Adjust size and margin of visibility toggles for both CD8+ percentile and TMB information

### CODE REVIEW

File: app/views/ReportView/components/RapidSummary/index.tsx [hunk 1]
```diff
@@ -376,7 +386,7 @@ const RapidSummary = ({
         value: msiStatus,
       },
     ]);
-  }, [microbial, primaryBurden, tmburMutBur, report.m1m2Score, report.sampleInfo, report.tumourContent, report.captiv8Score, tCellCd8]);
+  }, [microbial, primaryBurden, tmburMutBur, report.m1m2Score, report.sampleInfo, report.tumourContent, tCellCd8.percentile, tCellCd8.score, report.captiv8Score, tCellCd8.percentileHidden, tCellCd8]);
```
Comment (kttkjl) [inline app/views/ReportView/components/RapidSummary/index.tsx:389]: missing some dependencies? 

`tCellCd8.pedsScoreComment`  `tmburMutBur.adjustedTmb` `tmburMutBur.tmbHidden` `tCellCd8.pedsScore` `tCellCd8.pedsPercentile`

**Review by kttkjl — CHANGES_REQUESTED** [review]
Works as intended, missing some dependencies in react hooks

File: app/views/ReportView/components/RapidSummary/index.tsx (hunk 1 above)
Comment (bnguyen-bcgsc) [reply app/views/ReportView/components/RapidSummary/index.tsx:389]: oops fixing it now

**Review by kttkjl — APPROVED** [review]
Would it be possible to put the comment for a commit be a short description first, and then the ticket number at the very end? on mouseover for the commits tells nothing of the commit at hand and it shows up as `- 2244` instead of a description

something like 

```
feat: added some salt to the chicken 

- KFC-420
```

**Pull request merged**